<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MailOpenWindow</class>
 <widget class="QDialog" name="MailOpenWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Open Mail</string>
  </property>
  <property name="windowIcon">
   <iconset resource="general.qrc">
    <normaloff>:/images/potter.svg</normaloff>:/images/potter.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
	border-radius:15px;
	background-color:#fa2569;
	color:white;
}

QPushButton:hover
{
	border-radius:15px;
	background-color:#feb83d;
	color:black;
}

QLabel
{
	background-color:rgba(0, 0, 0,0);
	border: 1px solid grey;
	border-color:#feb83d;
	border-radius: 5px;
	color:white;
}

QRadioButton
{
	border: 1px solid grey;
	border-color:#feb83d;
	border-radius: 5px;
	color:#fa2569;
}

QLineEdit
{
	background-color:rgba(0, 0, 0,0);
	border: 1px solid grey;
	border-color:#feb83d;
	border-radius: 5px;
	color:white;
}

QMessageBox 
{
    background-color: #220c28;
}

QMessageBox QLabel 
{
    	background-color:rgba(0, 0, 0,0);
	border: 1px solid grey;
	border-color:#220c28;
	border-radius: 5px;
	color:white;
}

QMessageBox QPushButton
{
	border-radius:15px;
	border: 1px solid grey;
	border-color:#fa2569;
	background-color:#220c28;
	color:white;
}

QMessageBox QPushButton::hover
{
	border-radius:15px;
	border: 1px solid grey;
	border-color:#feb83d;
	background-color:#220c28;
	color:white;
}

QTextBrowser
{
	background-color:rgba(0, 0, 0,0);
	border: 1px solid grey;
	border-color:#feb83d;
	border-radius: 5px;
	color:white;
}

#MailOpenWindow
{
	background-color:#220c28;
}
</string>
  </property>
  <widget class="QLabel" name="from_text_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="to_text_label">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="from_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>FROM:</string>
   </property>
  </widget>
  <widget class="QLabel" name="to_label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TO:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="read_textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>761</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="date_label">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>20</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>DATE:</string>
   </property>
  </widget>
  <widget class="QLabel" name="date_text_label">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="general.qrc"/>
 </resources>
 <connections/>
</ui>
